sim/deathknight/scourge_strike.go

```diff
@@ -76,6 +76,8 @@ func (dk *Deathknight) registerScourgeStrikeSpell() {
 					if dk.DiseasesAreActive(spellEffect.Target) {
 						dk.LastScourgeStrikeDamage = spellEffect.Damage
 						shadowDamageSpell.Cast(sim, spellEffect.Target)
+						shadowDamageSpell.SpellMetrics[spellEffect.Target.Index].Casts--
+						shadowDamageSpell.SpellMetrics[spellEffect.Target.Index].Hits--
```

Comment:
@lologarithm why was this done?

Reply:
![image](https://user-images.githubusercontent.com/5707996/181709383-5a9f7777-4216-45f8-bdb8-3e545beeff91.png)
 it breaks the shadow damage reporting in details and i think should be reverted